roles/ceph-mon/defaults/main.yml

```diff
@@ -0,0 +1,26 @@
+---
+# You can override vars by using host or group vars
+
+###########
+# GENERAL #
+###########
+
+# ACTIVATE BOTH FSID AND MONITOR_SECRET VARIABLES FOR NON-VAGRANT DEPLOYMENT
+fsid: "{{ cluster_uuid.stdout }}"

```

Comment:
Using the above namespace convention will invalidate the use of group_vars (I think it is easier to define variable on a single file rather than separate roles).

I guess we aren't using it in ursula?